site/docs/tools/iceberg/spark.md

```diff
@@ -300,7 +166,6 @@ follows:
     ```
 === "Python"
     ``` python
-    # same code as the spark2 section above to create the testing.region table
```

Comment:
I think the above spark2 section had detailed steps for java and python. We might have to move/update that code blocks to spark3 section.

I am talking about line 165-283